Library/Homebrew/caveats.rb

```diff
@@ -44,8 +44,17 @@ def keg
   def keg_only_text
     return unless f.keg_only?
 
-    s = "This formula is keg-only, which means it was not symlinked into #{HOMEBREW_PREFIX}."
-    s << "\n\n#{f.keg_only_reason}\n"
+    reason = f.keg_only_reason.to_s
+    # Force lowercase for style consistency unless a name/brand/etc.
+    formal_names = "Apple|OS|Xcode|CLT|#{f.name[0].upcase}"
```

Comment:
Fair enough 👍. I thought you'd likely feel that way but wasn't any harm in playing with it this way & seeing how you felt 😸